### DISCUSSION TITLE
为什么勾了这个还是会把小逻撤下来进宿舍啊

### DISCUSSION THREAD

**Discussion by memgkong** [MaaAssistantArknights/MaaAssistantArknights#10660] [Q&A]
![image](https://github.com/user-attachments/assets/23cfe84c-9ede-4503-bb14-805134939e91)

**Comment by ABA2396** [comment]
日志

**Reply by memgkong** [reply]
[debug.zip](https://github.com/user-attachments/files/17097460/debug.zip)
是这些吗，我不知道是哪个

**Reply by ABA2396** [reply]
https://github.com/MaaAssistantArknights/MaaAssistantArknights/issues/new?template=cn-bug-report.yaml
